docker-compose.yml

```diff
@@ -34,11 +34,13 @@ services:
     environment:
       - DEBUG=${DEBUG-}
       - DOCKER_HOST=unix:///var/run/docker.sock
+      - HOSTNAME_EXTERNAL=localhost.localstack.cloud
```

Comment:
Can you elaborate on the reasoning behind this? It looks like the LocalStack [default](https://docs.localstack.cloud/references/configuration/) is `localhost` – wouldn't this require a LocalStack cloud account?

Reply:
This shouldn't have made it into my non-draft PR, will remove!